spring-boot-project/spring-boot/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```
@@ -105,7 +105,7 @@
     {
       "name": "logging.file.max-history",
       "type": "java.lang.Integer",
-      "description": "Maximum of archive log files to keep. Only supported with the default logback setup.",
+      "description": "Maximum of retention days the log archive log files to keep. Only supported with the default logback setup.",
```

What about using this description?

```suggestion
      "description": "Maximum number of days, archive log files are kept. Only supported with the default logback setup.",
```